plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/render/displayAdapter/impl/ViewportPaneSWT.java

```diff
@@ -318,6 +349,8 @@ private boolean clipContainsRepaintRequest( Rectangle clipping ) {
             disposeMutex = null;
 
             swtImage = createImage();
+            this.swtImageTrsf=renderManager.getMapInternal().getViewportModel().worldToScreenTransform();
```

Comment:
Without any check for null. I guess ViewportModel might be null during startup of bundles